target/riscv64.mk

```diff
@@ -16,6 +16,11 @@ CXX := $(CROSS)g++
 OLVL ?= -O2
 CFLAGS += -fomit-frame-pointer -mcmodel=medany
 
+ifneq ($(RISCV_ISA_STRING),)
+  CFLAGS += -march=$(RISCV_ISA_STRING)
+endif
+# else use toolchain defaults
```

Comment:
I'm on the fence regarding default - might be error prone, because it's not easy to notice that the default is used instead of desired one. I think it would be better to require it always. @kber-ps  your thoughts?

Reply:
Ok, might be better to always define it explicitly